src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Services/MethodologyServiceTests.cs

```diff
@@ -0,0 +1,346 @@
+                        Status = Approved,
+                        Title = "Methodology 1 title",
+                        Version = 0
+                    },
+                    new Methodology
+                    {
+                        Id = Guid.Parse("926750dc-b079-4acb-a6a2-71b550920e81"),
+                        Annexes = new List<ContentSection>(),
+                        Content = new List<ContentSection>(),
+                        PreviousVersionId = Guid.Parse("7a2179a3-16a2-4eff-9be4-5a281d901213"),
+                        PublishingStrategy = Immediately,
+                        Slug = slug,
+                        Status = Approved,
+                        Title = "Methodology 1 title updated",
+                        Version = 1
+                    },
+                    // Latest version is a draft that should not be returned
+                    new Methodology
+                    {
+                        Id = Guid.Parse("ad13999c-4caf-4f82-8df0-94db22acbcbc"),
+                        Annexes = new List<ContentSection>(),
+                        Content = new List<ContentSection>(),
+                        PreviousVersionId = Guid.Parse("926750dc-b079-4acb-a6a2-71b550920e81"),
+                        PublishingStrategy = Immediately,
+                        Slug = slug,
+                        Status = Draft,
+                        Title = "Methodology 1 title draft",
+                        Version = 2
+                    }
+                )
+            };
+
+            // Set up a different methodology with a different slug to make sure it's not returned
+            var methodology2 = new MethodologyParent
+            {
+                Slug = "methodology-2",
+                Versions = AsList(
+                    new Methodology
+                    {
+                        Annexes = new List<ContentSection>(),
+                        Content = new List<ContentSection>(),
+                        PreviousVersionId = null,
+                        PublishingStrategy = Immediately,
+                        Slug = "methodology-2",
+                        Status = Approved,
+                        Title = "Methodology 2 title",
+                        Version = 0
+                    }
+                )
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                await contentDbContext.MethodologyParents.AddRangeAsync(methodology1, methodology2);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                var service = SetupMethodologyServiceTests(contentDbContext: contentDbContext);
+
+                var result = await service.Get(slug);
+
+                Assert.True(result.IsRight);
```

Comment:
You can also use:

```
var methodology = result.AssertRight();
```

as a short-hand if you want?  Hot off the press!

Reply:
Thanks I saw that incoming change and was meaning to do this but forgot! I've changed this test class to use this now 🙂 .